GVFS/GVFS.Common/Git/GitAuthentication.cs

```diff
@@ -18,6 +18,8 @@ public class GitAuthentication
         private DateTime lastAuthAttempt = DateTime.MinValue;

```

Comment:
These changes could also be pulled out into a separate PR (or into the change series that leverages it)... just a thought